hugegraph-llm/src/hugegraph_llm/models/embeddings/litellm.py

```diff
@@ -77,17 +77,17 @@ def get_texts_embeddings(self, texts: List[str]) -> List[List[float]]:
             log.error("Error in LiteLLM batch embedding call: %s", e)
             raise
```

Comment:
Docstring says "single text" but function returns a batch. Please update docstring and add retry/backoff for async path similar to sync get_texts_embeddings (tenacity AsyncRetrying or equivalent). Also consider configurable max batch size and concurrency limits.